### PR TITLE
Amp/add rake tasks issue 15

### PR DESCRIPTION
**Pull request by amarkpark**
Added usertasks rake file and updated Readme with my name and heroku link.

### CODE REVIEW

File: lib/tasks/usertasks.rake [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
+# usertasks.rake
```
Comment (kenmazaika) [inline lib/tasks/usertasks.rake]: @amarkpark thoughts on adding a note how to use these scripts on the `README.md` file?

**Comment by kenmazaika** [comment]
looks good.  made 1 comment though.

File: lib/tasks/usertasks.rake (hunk 1 above)
Comment (amarkpark) [inline lib/tasks/usertasks.rake]: Will do

**Comment by amarkpark** [comment]
Updated readme with use instructions is good to go.

File: lib/tasks/usertasks.rake (hunk 1 above)
Comment (kenmazaika) [inline lib/tasks/usertasks.rake]: Awesome!  Thanks! :) :+1:

**Pull request merged**